apps/main/src/dex/entities/appstats-volume.ts

```diff
@@ -1,13 +1,12 @@
-import type { CurveApi } from '@/dex/types/main.types'
+import type { ChainId } from '@/dex/types/main.types'
 import { queryFactory } from '@ui-kit/lib/model/query'
-import { EmptyValidationSuite } from '@ui-kit/lib'
 import curvejsApi from '@/dex/lib/curvejs'
 import useStore from '@/dex/store/useStore'
+import { curvejsValidationSuite } from './validation/curvejs-validation'
 
-async function _fetchAppStatsVolume({ curve }: { curve: CurveApi | null }) {
-  if (!curve) return null
-
-  const chainId = curve.chainId
+async function _fetchAppStatsVolume({ chainId }: { chainId: ChainId | null }) {
```

Comment:
The query shouldn't get a nullable. 
This should be covered by validation suite too.